projects/ngx-d3timeline/src/lib/content/activity-drag.service.ts

```diff
@@ -1,37 +1,27 @@
 import { Injectable } from '@angular/core';
-import { tempStateSelector } from '../store/timeline-selectors';
 import { BehaviorSubject } from 'rxjs';
 import { Store } from '../store/store';
 import { withLatestFrom, filter } from 'rxjs/operators';
-import { getDropActivity } from '../drag-utils';
 import { drag } from 'd3-drag';
 import { select, event } from 'd3-selection';
 
 import * as fromActions from '../store/actions';
-import { ActivityRectangle } from './activity-rectangle';
 import { identifier } from '../types';
+import { selectDraggedToActivity } from './selectors/activity.selectors';
 
 @Injectable({ providedIn: 'root' })
 export class ActivityDragService {
   private dragEndSubject = new BehaviorSubject(null);
 
   drop$ = this.dragEndSubject.pipe(
```

Comment:
something like `droppedActivity$`  or `droppedUpdatedActivity$` ? a bit hard to see exactly what this is to name it.

Reply:
done